randomizer/Lists/CustomLocations.py

```diff
@@ -2755,7 +2755,7 @@ class LocationTypes(IntEnum):
             z=1159,
             max_size=64,
             logic_region=Regions.TrainingGrounds,
-            logic=lambda l: l.HasGun(Kongs.any) and (l.istiny and l.settings.krusha_kong == Kongs.tiny) or (not l.istiny) and l.swim,
+            logic=lambda l: (l.HasGun(Kongs.any) and (l.istiny and l.settings.krusha_kong == Kongs.tiny) or not l.istiny) and l.swim and l.scope,
```

Comment:
`(l.HasGun(Kongs.any) and ` -> `l.HasGun(Kongs.any) and (`
Same for the other training grounds crate. Otherwise it can mean "you need to have any gun except tiny's, or just don't be tiny and you can do it moveless.